var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -122,6 +122,10 @@ class Conduit(CMakePackage):
     depends_on("hdf5@1.8.19:1.8~shared~cxx", when="+hdf5+hdf5_compat~shared")
     depends_on("hdf5~cxx", when="+hdf5~hdf5_compat+shared")
     depends_on("hdf5~shared~cxx", when="+hdf5~hdf5_compat~shared")
+
+    # conduit uses a <1.10 api version.
+    depends_on("hdf5@:1.10", when="+hdf5")
```

Comment:
attn @cyrush: is Conduit incompatible with the HDF5 1.12 and 1.13 release lines?
Might need upstream tracking.

Reply:
conduit 0.8.0 is compat with HDF5 1.12 (I haven't tried 1.13 line). I am doing an update to the conduit package for 0.8.0 -- let me add that change.

Reply:
sorry I was confused. We don't need this line with conduit 0.8.0 +

Reply:
```suggestion
    depends_on("hdf5@:1.10", when="@:0.7 +hdf5")
```